src/monodroid/jni/monodroid-glue.cc

```diff
@@ -1635,6 +1651,19 @@ MonodroidRuntime::Java_mono_android_Runtime_initInternal (JNIEnv *env, jclass kl
 		total_time.mark_start ();
 	}
 
+#if defined (NET6)
+	{
+		PInvokeOverrideFn pinvoke_override_cb = monodroid_pinvoke_override;
+		char ptr_str[20];
+		snprintf (ptr_str, sizeof(ptr_str), "0x%p", pinvoke_override_cb);
+
+		const char* property_keys[] { "PINVOKE_OVERRIDE" };
+		const char* property_values[] { ptr_str };
```

Comment:
this remains gloriously insane.

Reply:
It is... Thus, https://github.com/dotnet/runtime/issues/48416 :)